### ISSUE TITLE
[field] accessibility: Example fails non-text contrast requirement

### ISSUE THREAD

**Issue by grace-snow** [mui/base-ui#4252]
# Bug report

## Current behavior

The input in the [Field component example]() is very difficult to see because the input border has such low contrast against adjacent colours. As a result, this fails [WCAG 2.2 Success Criterion 1.4.11 Non-text Contrast](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)

## Expected behavior

The border has a near-transparent colour (oklch(0.12 0.04 264.01 / 0.07)) but must have at least a 3:1 contrast ratio against adjacent colours. This is the requirement for all UI components and meaningful graphics.

## Reproducible example

1. Navigate to the [Field component example](https://base-ui.com/react/components/field)
2. Use a tool like the [Colour contrast analyser](https://vispero.com/color-contrast-checker/) to colour pick the border colour and adjacent background colour
3. Note the ratio 

## Base UI version

v1.2.0

## Which browser are you using?

Chrome

## Which OS are you using?

Mac OS

## Which assistive tech are you using (if applicable)?

N/A

## Additional context

Ensure all component examples meet contrast requirements to demonstrate best practices.

**Comment by aarongarciah** [comment]
We're working on some docs UI improvements that include revising all colors used in demos, so we'll likely tackle this in the upcoming weeks.

**Comment by aarongarciah** [comment]
This was fixed via https://github.com/mui/base-ui/pull/4676
